### PR TITLE
fix(pricing): add missing migrations for pricing module

### PR DESCRIPTION
**Pull request by riqwan**

### CODE REVIEW

**Review by pKorsholm — APPROVED** [review]
lgtm

**Review by fPolic — APPROVED** [review]
LGTM

File: packages/pricing/src/migrations/Migration20231101232834.ts [hunk 1]
```diff
@@ -44,6 +44,20 @@ export class Migration20231101232834 extends Migration {
...
+
+    this.addSql(
+      'alter table "money_amount" add column if not exists "created_at" timestamptz not null default now(), add column if not exists "updated_at" timestamptz not null default now(), add column if not exists "deleted_at" timestamptz null;'
+    )
+
+    this.addSql('drop index if exists "IDX_money_amount_deleted_at";')
+
+    this.addSql(
+      'create index "IDX_money_amount_deleted_at" on "money_amount" ("deleted_at");'
+    )
```
Comment (adrien2p) [inline packages/pricing/src/migrations/Migration20231101232834.ts]: question: any reason to drop and re create the index?

Comment (riqwan) [reply packages/pricing/src/migrations/Migration20231101232834.ts]: Just a safety mechanism, to always start from a clean slate. :)

Comment (adrien2p) [reply packages/pricing/src/migrations/Migration20231101232834.ts]: wouldn't a REINDEX be enough?

Comment (adrien2p) [reply packages/pricing/src/migrations/Migration20231101232834.ts]: something like IF NOT EXISTS maybe?

Comment (riqwan) [reply packages/pricing/src/migrations/Migration20231101232834.ts]: updated

**Pull request merged**